src/test/java/com/notably/logic/suggestion/SuggestionTestUtil.java

```diff
@@ -31,8 +31,10 @@ public static Model getModel() {
         return model;
     }
 
-    public static AbsolutePath getToCs2103t() {
-        return TypicalBlockModel.PATH_TO_CS2103T;
+    public static List<AbsolutePath> getToCs2103t() {
+        List<AbsolutePath> pathsToCs2103t = new ArrayList<>();
+        pathsToCs2103t.add(TypicalBlockModel.PATH_TO_CS2103T);
+        return pathsToCs2103t;
```

Comment:
Can use `List.of()` to make this more concise btw

Reply:
I'd rather make this return an `AbsolutePath` instance, then make it a list on the test cases themselves actually. This will eliminate the awkward `.get(0)` calls. But just my opinion!

Reply:
Will change, thanks @kevinputera !